styles/GlobalFonts.tsx

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import AppLoading from 'expo-app-loading';
+import {
+  useFonts,
+  PTSans_400Regular, // PTSans is Google's closest match to Myriad Pro
+  PTSans_400Regular_Italic,
+  PTSans_700Bold,
+  PTSans_700Bold_Italic,
+} from '@expo-google-fonts/pt-sans';
+
+function declareFonts() {
+  const [fontsLoaded] = useFonts({
+    PTSans_400Regular,
+    PTSans_400Regular_Italic,
+    PTSans_700Bold,
+    PTSans_700Bold_Italic,
+  });
+
+  if (!fontsLoaded) {
+    return <AppLoading />;
+  }
+  return <div />;
```

Comment:
I don't think the return value is being used by App.tsx

Reply:
Fixed by moving GlobalFonts logic into `App.tsx `

Reply:
moving this logic to App.tsx fixed issue!

Reply:
> Also, is it expected that the text appears at the top of each page instead of the middle?

Yes, I changed styling a bit to accommodate incoming features